iot/contiki/README.md

```diff
@@ -0,0 +1,165 @@
+The Contiki Operating System for IoT
+====================================
+
+Contiki is an open source operating system that runs on tiny low-power
+microcontrollers and makes it possible to develop applications that
+make efficient use of the hardware while providing standardized
+low-power wireless communication for a range of hardware platforms.
+
+Contiki is used in numerous commercial and non-commercial systems,
+such as city sound monitoring, street lights, networked electrical
+power meters, industrial monitoring, radiation monitoring,
+construction site monitoring, alarm systems, remote house monitoring,
+and so on.
+
+For more information, see the Contiki website [here](http://contiki-os.org)
```

Comment:
I would add a `.` at the end of `...Contiki website [here](http://contiki-os.org)`, so
```
For more information, see the Contiki website [here](http://contiki-os.org).
```

<br>

BTW, the hyperlink will still be there if you just do: `http://contiki-os.org`, like this: http://contiki-os.org.
But, how you changed it is better :)